vipps-recurring-api.md

```diff
@@ -166,6 +167,38 @@ These headers are required for plugins and partners and sent by the recent versi
 [the official Vipps plugins](https://github.com/vippsas/vipps-developers#plugins)
 and we recommend all customers with direct integration with the API to also do so.
 
+## orderId recommendations
+
+A `orderId` must be unique. 
+
+If you ever have a problem that requires us to search in our logs, we need
+`orderId`s that are "unique enough" to actually find them. An `orderId` that
+is just a number may not be possible to find.
+
+While the minimum length for `orderId` _technically_ is just one character,
+we strongly recommend using at least 6 characters, and a combination of numbers
+and characters.
+
+The maximum length of an `orderId` is 50 alphanumeric characters:
```

Comment:
pretty sure the max-length in recurring is 40 characters, because we were unsure if any characters we allowed would be translated to two or more characters in DWO etc (like _ becomes UL), so we wanted a little buffer

Reply:
Thanks, changed to 40.